python/tests/res/fm/test_ecl_run.py

```diff
@@ -94,84 +94,80 @@ def init_flow_config(self):
+        ecl_config = Ecl100Config()
+        sim = ecl_config.sim("2014.2")
+        mpi_sim = ecl_config.mpi_sim("2014.2")
+        ecl_run = EclRun("ECLIPSE.DATA", sim)
+        self.assertEqual( ecl_run.runPath() , os.getcwd())
 
-            os.mkdir("path")
-            with open("path/ECLIPSE.DATA" , "w") as f:
-                f.write("Mock eclipse data file")
+        os.mkdir("path")
+        with open("path/ECLIPSE.DATA" , "w") as f:
+            f.write("Mock eclipse data file")
 
-            ecl_run = EclRun("path/ECLIPSE.DATA", sim)
-            self.assertEqual( ecl_run.runPath() , os.path.join(os.getcwd() , "path"))
-            self.assertEqual( ecl_run.baseName() , "ECLIPSE")
-            self.assertEqual( 1 , ecl_run.numCpu())
+        ecl_run = EclRun("path/ECLIPSE.DATA", sim)
+        self.assertEqual(ecl_run.runPath() , os.path.join(os.getcwd() , "path"))
+        self.assertEqual(ecl_run.baseName() , "ECLIPSE")
+        self.assertEqual(1, ecl_run.numCpu())
 
-            # invalid number of CPU
-            with self.assertRaises(ValueError):
-                ecl_run = EclRun("path/ECLIPSE.DATA", sim, num_cpu = "xxx")
+        # invalid number of CPU
+        with self.assertRaises(ValueError):
+            ecl_run = EclRun("path/ECLIPSE.DATA", sim, num_cpu = "xxx")
 
-            # invalid number of CPU
-            with self.assertRaises(Exception):
-                ecl_run = EclRun("path/ECLIPSE.DATA", sim, num_cpu = 10)
+        # invalid number of CPU
+        with self.assertRaises(Exception):
+            ecl_run = EclRun("path/ECLIPSE.DATA", sim, num_cpu = 10)
 
-            ecl_run = EclRun("path/ECLIPSE.DATA", mpi_sim, num_cpu = "10")
-            self.assertEqual( 10 , ecl_run.numCpu())
+        ecl_run = EclRun("path/ECLIPSE.DATA", mpi_sim, num_cpu = "10")
+        self.assertEqual( 10 , ecl_run.numCpu())
 
-            #Missing datafile
-            with self.assertRaises(IOError):
-                ecl_run = EclRun("DOES/NOT/EXIST", mpi_sim, num_cpu = "10")
+        #Missing datafile
+        with self.assertRaises(IOError):
+            ecl_run = EclRun("DOES/NOT/EXIST", mpi_sim, num_cpu = "10")
 
     @pytest.mark.xfail(reason="Finding a version on Komodo of flow that is not OPM-flow")
     @flow_installed
     def test_flow(self):
-        with TestAreaContext("ecl_run") as ta:
-            self.init_flow_config()
-            ta.copy_file( os.path.join(self.SOURCE_ROOT , "test-data/local/eclipse/SPE1.DATA"))
-            ta.copy_file( os.path.join(self.SOURCE_ROOT , "test-data/local/eclipse/SPE1_ERROR.DATA"))
-            os.makedirs("ecl_run")
-            shutil.move("SPE1.DATA", "ecl_run")
-            shutil.move("SPE1_ERROR.DATA" , "ecl_run")
-            flow_config = FlowConfig()
-            sim = flow_config.sim()
-            flow_run = EclRun("ecl_run/SPE1.DATA", sim)
-            flow_run.runEclipse( )
+        self.init_flow_config()
```

Comment:
I suppose this test should get the `use_tmpdir` fixture as well, right?

Reply:
Yes, which is in #727, though we might consider rewriting the whole tmpdir implementation somewhat based on feedback in that PR.

Reply:
I am not sure I follow, in #727 `test_ecl_run.py` is not touched

Reply:
You're right, I misread your comment, I've removed TestAreaContext from test_flow